### PR TITLE
Update the formatting of tooltips

### PR DESCRIPTION
**Pull request by danchurch-wk**
https://github.com/Workiva/lsif_indexer/issues/22

## Problem
Sourcegraph doesn't seem to handle the `markdown` content kind, so newlines are not respected

## Solution
Style all of the hover tooltip as `Dart` code, include:
  - Package the member comes from (except if declared in the same file)
  - The definition of the member
  - The docstring associated to the member (if one exists)

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: lib/src/graph/identifier.dart [hunk 1]
```diff
@@ -183,16 +180,37 @@ class LocalDeclaration extends Identifier implements AbstractDeclaration {
...
-        'result': {
-          'contents': {'kind': 'markdown', 'value': hoverText}
-        }
-      };
+  Map<String, Object> toLsif() {
+    return {
+      ...super.toLsif(),
+      'result': {
+        'contents': [package, declaration, docString]
+            .whereNotNull()
```
Comment (alanknight-wk) [inline lib/src/graph/identifier.dart:208]: nit: you could do this as 
`   ... get _hoverTexts [package, declaration, docString];`
then
`  'contents': [for (var text in _hoverTexts) if (text != null) {'language': 'dart', 'value': text}];`
which feels a bit nicer to me - but I really like the collection literal stuff. And then you wouldn't need the extension method.

**Review by alanknight-wk — DISMISSED** [review]

**Review by alanknight-wk — APPROVED** [review]

**Review by alanknight-wk — APPROVED** [review]

**Comment by danchurch-wk** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**